### PR TITLE
[modular][ready]a full engineer ert and...one for plants 

### PR DESCRIPTION
**Pull request by thestubborn**
## About The Pull Request

title

## Why It's Good For The Game

first one was a request, latter was because of THE WEEDS

## Changelog
:cl:
add: a full engineer ert and one for plants 
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/additionalert/fumigators/code/modules/antagonists/ert/ert.dm [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/datum/outfit/centcom/ert/weedkiller
+	name = "Fumigator"
+	outfit = /datum/outfit/centcom/ert/pizza
```
Comment (tf-4) [inline modular_skyrat/modules/additionalert/fumigators/code/modules/antagonists/ert/ert.dm]: ?

File: modular_skyrat/modules/additionalert/fumigators/code/datums/ert.dm [hunk 2]
```diff
@@ -0,0 +1,7 @@
+/datum/ert/weedkiller
+	roles = list(/datum/antagonist/ert/pizza)
```
Comment (tf-4) [inline modular_skyrat/modules/additionalert/fumigators/code/datums/ert.dm]: ?

**Review by tf-4 — COMMENTED** [review]
Why is this pizza

File: modular_skyrat/modules/additionalert/fumigators/code/modules/antagonists/ert/ert.dm (hunk 1 above)
Comment (thestubborn) [reply modular_skyrat/modules/additionalert/fumigators/code/modules/antagonists/ert/ert.dm]: i forgot

File: modular_skyrat/modules/additionalert/fumigators/code/datums/ert.dm (hunk 2 above)
Comment (thestubborn) [reply modular_skyrat/modules/additionalert/fumigators/code/datums/ert.dm]: to edit it

**Comment by thestubborn** [comment]
> 
> 
> Why is this pizza

because I'm a moron and forgot to change it, fixed now

File: modular_skyrat/modules/additionalert/fumigators/code/datums/ert.dm [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+	leader_role = /datum/antagonist/ert/pizza/leader
```
Comment (FlamingLily) [inline modular_skyrat/modules/additionalert/fumigators/code/datums/ert.dm]:
```suggestion
	roles = list(/datum/antagonist/ert/weedkiller)
	leader_role = /datum/antagonist/ert/weedkiller/leader
```
These are still pizza

**Comment by Gandalf2k15** [comment]
yes

**Comment by tf-4** [comment]
Doesn't plant ERT already exist in the form of janitor ERT?

**Comment by thestubborn** [comment]
> 
> 
> Doesn't plant ERT already exist in the form of janitor ERT?

i don't think they get stuff to kill weeds
(I was p close to giving em flame throwers honestly)

**Pull request merged**